src/zos.cc

```diff
@@ -2674,10 +2715,20 @@ int __zinit::initialize(const zoslib_config_t &aconfig) {
     setenv("_EDC_SUSV3", "1", 1);
   }
 
-  tenv = getenv("_BPXK_AUTOCVT");
-  if (!tenv || !*tenv) {
-    setenv("_BPXK_AUTOCVT", "ON", 1);
+  std::string runopts;
+  if (get_env_var("_CEE_RUNOPTS", runopts)) {
+    std::ostringstream value_ostr;
+    value_ostr << runopts << " FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)";
```

Comment:
should there be a check if either of these is already specified? If _BPX_AUTOCVT=OFF is specified, don't you need to override it?